pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/CalmRules.scala

```diff
@@ -23,7 +23,7 @@ trait UseCalmWhenExistsRule extends FieldMergeRule with MergerLogging {
     sources: Seq[TransformedBaseWork]): FieldMergeResult[FieldData] =
     FieldMergeResult(
       fieldData = calmRule(target, sources) getOrElse getData(target),
-      redirects = Nil
+      mergedFrom = Nil
```

Comment:
Given the rule about "any" data, I don't think this being `Nil` (here and in other rules) makes sense any more?